src/lib/netlist/macro/nlm_ttl74xx_lib.cpp

```diff
@@ -3254,6 +3470,29 @@ static TRUTH_TABLE(TTL_74279B, 4, 1, "")
 	TT_FAMILY("74XX")
 }
 
+static TRUTH_TABLE(TTL_74368_GATE, 5, 4, "")
+{
+	TT_HEAD("OE,A1,A2,A3,A4|Y1,Y2,Y3,Y4")
+	TT_LINE("1,X,X,X,X|0,0,0,0|45")
+	TT_LINE("0,0,0,0,0|1,1,1,1|45")
+	TT_LINE("0,0,0,0,1|1,1,1,0|45")
+	TT_LINE("0,0,0,1,0|1,1,0,1|45")
+	TT_LINE("0,0,0,1,1|1,1,0,0|45")
+	TT_LINE("0,0,1,0,0|1,0,1,1|45")
+	TT_LINE("0,0,1,0,1|1,0,1,0|45")
+	TT_LINE("0,0,1,1,0|1,0,0,1|45")
+	TT_LINE("0,0,1,1,1|1,0,0,0|45")
+	TT_LINE("0,1,0,0,0|0,1,1,1|45")
+	TT_LINE("0,1,0,0,1|0,1,1,0|45")
+	TT_LINE("0,1,0,1,0|0,1,0,1|45")
+	TT_LINE("0,1,0,1,1|0,1,0,0|45")
+	TT_LINE("0,1,1,0,0|0,0,1,1|45")
+	TT_LINE("0,1,1,0,1|0,0,1,0|45")
+	TT_LINE("0,1,1,1,0|0,0,0,1|45")
+	TT_LINE("0,1,1,1,1|0,0,0,0|45")
+	TT_FAMILY("74XX")
```

Comment:
These propagation delays seem unreasonably long.  Typical propagation delays are below 10ns for high-to-low and vice versa, and below 25ns to exit high-impedance state.  Worst case for entering high-impedance state is quoted at 35ns.  The propagation delay of 45ns here is worse than the worst case in the datasheet for any transition.